client/components/button/docs/example.jsx

```diff
@@ -23,19 +25,38 @@ var Buttons = React.createClass( {
 	},
 
 	render: function() {
-		var toggleButtonsText = this.state.compactButtons ? 'Normal Buttons' : 'Compact Buttons';
+		var toggleText = this.state.compactButtons ? 'Normal Buttons' : 'Compact Buttons';
+		return config.isEnabled( 'devdocs/usage-stats' )
+			? this.renderNewStructure( toggleText )
```

Comment:
Don’t forget about “off-by-one errors” :-)

The new names seem a lot clearer to me: `renderDocsExample` and `this.renderDocsExampleWithUsageStats`.